.github/workflows/create-gui-pr.yml

```diff
@@ -79,14 +75,14 @@ jobs:
           token: ${{ steps.generate-token.outputs.token }}
           path: main-application
           commit-message: |
-            chore(gui): updates GUI
+            chore(deps): bump kuma-gui to ${{ github.sha }}
 
-            Updates GUI.
+            Bumps kuma-gui to version ${{ github.ref_name }}@${{ github.sha }}
           committer: GitHub <noreply@github.com>
           author: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
           signoff: true
-          branch: chore/update-gui-${{ steps.date.outputs.date }}
+          branch: chore/update-gui-${{ github.sha }}
```

Comment:
**Suggestion**: Re-using the same branch will update the PR so we can avoid to close and open new PRs if things get merged in Kuma GUI before the PR in Kuma is merged.

```suggestion
          branch: chore/update-gui
```